code/linkfield/Extensions/LinkPageExtension.php

```diff
@@ -38,7 +38,7 @@ class LinkPageExtension extends Extension
 
     protected function updateCMSFields(FieldList $fields)
     {
-        $fields->removeByName(['Content', 'HasOneLinkID', 'HasManyLinksID']);
+        $fields->removeByName(['Content']);
```

Comment:
Formfields aren't autoscaffolded for `SiteTree` which this extension is made for.